src/integration_runner/main.go

```diff
@@ -178,6 +179,9 @@ func catRequest(w http.ResponseWriter, r *http.Request) {
 	env := merge(ctx.Env, nil)
 	settings := merge(ctx.Settings, nil)
 	settings["newrelic.appname"] = "ignore"
+	if ("false" == dtEnabled) {
+	    settings["newrelic.distributed_tracing_enabled"] = "false";
+	}
```

Comment:
@mfulb Do you mean add `else` with `settings["newrelic.distributed_tracing_enabled"] = "true";` if `dtEnabled` is anything different than false to explicitly enable distributed tracing and not assume default behavior?

Reply:
Hmm that is a good question @misiekn.  I was thinking more if the value of `dtEnabled` is a valid boolean then honor what it is asking for.  Otherwise ignore with a warning?

Reply:
@mfulb See [b0b06a8](https://github.com/newrelic/newrelic-php-agent/pull/386/commits/b0b06a8ee5b4af2d5e34334cc38873dcb2e12437) and let me know if it looks good.

Reply:
That looks good - I like having an error so it doesn't go potentially unnoticed if an improper value is passed.